include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdl_cshuffle_streamk_v3.hpp

```diff
@@ -555,6 +562,7 @@ struct GridwiseGemm_xdl_cshuffle_streamk_v3
         index_t StrideC;
         index_t Streamk_sel;
         mutable index_t Grid_size;
+        StreamKReductionStrategy reduction_strategy; // Added member variable
```

Comment:
What is the purpose of introducing this comment?
```suggestion
        StreamKReductionStrategy reduction_strategy;
```